recipes/ptex/all/patches/0001-fix-cmake.patch

```diff
@@ -0,0 +1,57 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -33,6 +33,10 @@ enable_testing()
+ # Setup platform-specific threading flags.
+ find_package(Threads REQUIRED)
+ 
++find_package(ZLIB REQUIRED)
++add_library(PkgConfig::Ptex_ZLIB INTERFACE IMPORTED)
++set_property(TARGET PkgConfig::Ptex_ZLIB PROPERTY INTERFACE_LINK_LIBRARIES ZLIB::ZLIB)
++if(0)
+ # Use pkg-config to create a PkgConfig::Ptex_ZLIB imported target
+ find_package(PkgConfig REQUIRED)
+ pkg_checK_modules(Ptex_ZLIB REQUIRED zlib IMPORTED_TARGET)
```

Comment:
Doesn't the `pkgconfig` generator + `pkgconf` as build requirement work?

Reply:
Sure, but I prefer to avoid pkgconf in build requirements, it's fragile on Macos if shared.